app/assets/src/api/index.js

```diff
@@ -4,7 +4,6 @@
 import axios from "axios";
 import axiosRetry from "axios-retry";
 
-import { cleanFilePath } from "~utils/sample";
```

Comment:
Should we delete cleanFilePath?

Reply:
This function isn't actually deleted, just removed from this file.

Reply:
I know, but it is not used anymore! So we should delete it.